style/ruby/README.md

```diff
@@ -1,93 +1,4 @@
 Ruby
 ====
 
-[Sample](sample.rb)
-
-* Avoid conditional modifiers (lines that end with conditionals). [36491dbb9]
-* Avoid multiple assignments per line (`one, two = 1, 2`). [#109]
-* Avoid organizational comments (`# Validations`). [#63]
-* Avoid ternary operators (`boolean ? true : false`). Use multi-line `if`
-  instead to emphasize code branches. [36491dbb9]
-* Avoid explicit return statements. [#48]
-* Avoid using semicolons. [#108]
-* Avoid bang (!) method names. Prefer descriptive names. [#122]
-* Don't include spaces after `(`, `[` or before `]`, `)`.
-* Don't use `self` explicitly anywhere except class methods (`def self.method`)
-  and assignments (`self.attribute =`). [#89]
-* Don't vertically align tokens on consecutive lines.
-* If you break up a chain of method invocations, keep each method invocation on
-  its own line. Place the `.` at the end of each line, except the last.
-* If you break up a hash, keep the elements on their own lines and closing curly
-  brace on its own line.
-* Indent continued lines two spaces.
-* Indent private methods equal to public methods.
-* Name variables created by a factory after the factory (`user_factory`
```

Comment:
Some of the things in here are ways for us to avoid thinking about things. Is it reasonable to preserve them?

Reply:
I am open to discussing specific things that we should keep if folks feel strongly. If folks to not feel strongly, I plan to remove them all.

For "Name variables created by a factory after the factory" specifically, I am OK with variables names not matching the name of the factory as long as the variable names are contextually meaningful.

Reply:
I don't feel particularly strongly about this rule, but I think it makes sense for a “use standard” commit to only remove the things that are enforced by or overridden by standard. If we want to also remove this rule, it should be in another commit to make the reasons clear.

Reply:
OK, I won't delete everything. 😢